### ISSUE TITLE
the photos could have metadata about where face is, used for cropping

### ISSUE THREAD

**Issue by mkf** [alexandersandberg/placekeanu.com#2]
the photos could have metadata about where face is, used for better cropping

additional routines could be written to do viewbox differently when there is such info attached for a given photo

![2019-07-22-123619_1016x244_scrot](https://user-images.githubusercontent.com/7753506/61626462-a45da200-ac7d-11e9-92b8-2d057dfd730d.png)

ofc iknow iknow that's a facultative feature and without it it's all still great

and i guess it was probably totally a conscious decision during the development, not to do that at the time

i hope it's not too bad that i just, maybe too boldly, put the idea here and not contribute doing it myself (since i guess i would have some chances in doing it with enough effort but rn i just don't feel like doing it, also i'm not a user of placekeanu really myself although i know people who totally are gonna be using it)

i mostly just do it thinking of people who might be stumbling into issues and maybe will be eager to do it

**Comment by alexandersandberg** [comment]
@mikrfe You're right—the cropping could be improved. At the moment it's only looking for a photo of the correct rotation when the width/height ratio is too far off from a square while centering everything width- and height wise.

I appreciate the suggestion! I'll keep the issue open as it's something I—or possibly someone else—would consider working on in the future.
